src/NuGet.Core/NuGet.ProjectModel/JsonPackageSpecReader.cs

```diff
@@ -27,6 +27,7 @@ public class JsonPackageSpecReader
         public static readonly string PackOptions = "packOptions";
         public static readonly string PackageType = "packageType";
         public static readonly string Files = "files";
+        public static readonly FileShare share = FileShare.ReadWrite | FileShare.Delete;
```

Comment:
I thought `share` was also self explanatory but I'm open to any good suggestion which you guys feel  would be more appropriate...